spec/index.bs

```diff
@@ -1920,6 +1922,29 @@ interface GPUAdapterInfo {
         this value is not recommended. Applications which change their behavior based on the
         {{GPUAdapterInfo}}, such as applying workarounds for known driver issues, should rely on the
         other fields when possible.
+
+    : <dfn>properties</dfn>
+    ::
+        {{GPUFeatureName|Feature-specific}} values of the [=adapter=].
+</dl>
+
+<script type=idl>
+interface GPUAdapterProperties {
+    readonly attribute unsigned long minSubgroupSize; // From "subgroups" feature
```

Comment:
nit: the comments don't help that much and would confuse users IMHO, can we reserve that to the description below?